cmd/config-manager/Dockerfile

```diff
@@ -1,19 +1,23 @@
-# Build Stage
 ARG GO_VERSION=1.25
 
-FROM golang:${GO_VERSION}-bookworm AS build
+FROM golang:${GO_VERSION}-bookworm AS builder
+
 WORKDIR /go/builder
 
 # Fetch go dependencies in a separate layer for caching
 COPY go.mod go.sum ./
 COPY pkg/topology/ pkg/topology/
 RUN go mod download
 
-# Build the config-manager
-COPY cmd/config-manager/*.go ./
-RUN go build -tags osusergo,netgo -ldflags "-extldflags=-static" -o config-manager
+# Build config-manager binary
+COPY . .
+
+RUN make clean install-go-licenses
+RUN make IMAGE_VERSION=${IMAGE_VERSION} BUILD_VERSION=${BUILD_VERSION} BUILD_BUILDID=${BUILD_BUILDID} PLUGINS= BINARIES=config-manager build-binaries-static licenses
```

Comment:
We have complex rules for licenses.% targets but use the top-level licenses target in the dockerfiles. Unintended?